recipes/ifaa/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ifaa" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ifaa-{{ version }}.tar.gz
+  sha256: fc3f148a91e8cb03d8690fd78c8f70d27ff20633e1db3a13eec9bf8c6aead166
+
+build:
+  number: 0
+  skip: True  # [not unix]
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+
+
+requirements:
+  build:
+    - python
+  host:
+    - pip
+    - python
+  run:
+    - glmnet_py
+    - joblib  
+    - matplotlib-base 
+    - numpy 
+    - pandas 
+    - python
+    - scipy
+    - statsmodels 
+    - tqdm 
+    
+test:
+  imports:
+    - ifaa
+    
+about:
+  home: https://github.com/lovestat/ifaa
+  summary: Robust association identification and inference for absolute abundance in microbiome analyses
+  license: MIT
```

Comment:
Upstream you mention different licenses, which one is correct?

1. https://github.com/lovestat/IFAA/blob/bd88b8f9344a4a01133781fe0c99b0a9f6c85931/setup.py#L6
2. https://github.com/lovestat/IFAA/blob/main/LICENSE.txt